### PR TITLE
Prl 434 hearing urgency

### PR DESCRIPTION
**Pull request by yogendraU**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
PRL-434

### Change description ###
Hearing Urgency screen refinement and move out as a separate event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by pete-lowe — DISMISSED** [review]

File: definitions/private-law/json/CaseField.json [hunk 1]
```diff
@@ -1278,5 +1174,69 @@
...
+    "ID": "CaseUrgencyTimeAndReason",
+    "Label": "Set out how soon the case needs to be heard (days/hours), and the reason for urgency.",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "EffortsMadeWithRespondents",
+    "Label": "What efforts have you made to put each respondent on notice of the application?",
```
Comment (pete-lowe) [inline definitions/private-law/json/CaseField.json:1198]: This doesn't match the copy on the figma screens.

File: definitions/private-law/json/CaseField.json [hunk 2]
```diff
@@ -1278,5 +1174,69 @@
...
+    "ID": "IsCaseUrgent",
+    "Label": "Is this case urgent?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "CaseUrgencyTimeAndReason",
+    "Label": "Set out how soon the case needs to be heard (days/hours), and the reason for urgency.",
```
Comment (pete-lowe) [inline definitions/private-law/json/CaseField.json:1190]: This doesn't match the copy on the figma screens - https://www.figma.com/file/M2WoOSgu8sRr0emyDWzf7Z/C100-Solicitor-prototype?node-id=1429%3A73092

File: definitions/private-law/json/CaseEventToFields.json [hunk 3]
```diff
@@ -1745,5 +1595,97 @@
...
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 2,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "C100",
+    "CaseEventID": "hearingUrgency",
+    "CaseFieldID": "EffortsMadeWithRespondents",
+    "DisplayContext": "OPTIONAL",
```
Comment (pete-lowe) [inline definitions/private-law/json/CaseEventToFields.json:1627]: I think this field should be mandatory (see the AC on the JIRA ticket)

File: definitions/private-law/json/CaseField.json (hunk 1 above)
Comment (pete-lowe) [reply definitions/private-law/json/CaseField.json]: updated the copy for label

File: definitions/private-law/json/CaseField.json (hunk 2 above)
Comment (pete-lowe) [reply definitions/private-law/json/CaseField.json]: updated the copy for label

**Review by pete-lowe — APPROVED** [review]

**Comment by yogendraU** [comment]
CLosing this PR and will create a fresh one as tests are failing.

**Pull request closed without merging**